fileUpload/index.php

```diff
@@ -138,36 +155,36 @@ function HumanSize($Bytes)
         }
     }
 
-	$result = $conn->query("select * from db_file;");
-	$countFiles = $result->num_rows;
+    $result = $conn->query("select * from db_file;");
+    $countFiles = $result->num_rows;
 
     ?>
 <div id="upload" style="width: 100%; height: 500px;position: relative">
 <div style="position: absolute; top: 0;width: 100%;">
-    <div class="container" style="">	
-		<div class="section">
-		<center><h1> FileUpload 176 </h1></center>	
-		    <form action="#" method="POST" enctype="multipart/form-data">
-		        <div class="file-field input-field" >
-		            <input class="file-path validate" type="text" style="width: 50%;"/>
-
-		            <div class="btn">
-		                <span>File</span>
-		                <input type="file" name="image"/>
-		            </div>
-		        </div>
-		        <div class="row">
-		            <div class="input-field col s12">
-		                <input name="url" id="url" type="url" class="validate">
-		                <label for="url">URL</label>
-		            </div>
-		        </div>
-		        <br/>
-		        <button type="submit" name="action" class="waves-effect waves-light btn" style="width:100%;">Upload</button>
-		    </form>
-		</div>
-		<br/>
-	</div>
+    <div class="container" style="">    
+        <div class="section">
+        <center><h1> FileUpload 176 </h1></center>  
+            <form action="#" method="POST" enctype="multipart/form-data">
+                <div class="file-field input-field" >
+                    <input class="file-path validate" id="inputFile" type="text" style="width: 50%;" onchange="validateField()"/>
+
+                    <div class="btn">
+                        <span>File</span>
+                        <input type="file" name="image"/>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="input-field col s12">
+                        <input name="url" id="url" type="url" class="validate" oninput="validateField()">
+                        <label for="url">URL</label>
+                    </div>
+                </div>
+                <br/>
+                <button id="uploadBtn" type="submit" name="action" class="waves-effect waves-light btn" style="width:100%;">Upload</button>

```

Comment:
<button id="uploadBtn" type="submit" name="action" class="waves-effect waves-light btn" style="width:100%;" disabled>Upload</button>   // Add this for inicialize disabled button.